### ISSUE TITLE
Make Origin Configurable. 

### ISSUE THREAD

**Issue by eldridgea** [timelinize/timelinize#52]
I'm running this in a hosted environment (the official docker container with Kubernetes + Cloudflare Tunnel) and serving it via a hostname I have. 

When loading the setup page I cannot select a file path for the repo and the browser console shows the error for the `/api` endpoint as `You can only access this API from a recognized origin`. It seems like the expected origin is `127.0.0.1` or `localhost`. Is there a way to configure this to accept a different origin? If not, could this be added? Possibly as an environment variable when running as a container.

**Comment by mholt** [comment]
Agreed! Good news is, this is planned. See #4. It's hard coded right now but I do need to make it configurable ASAP.

**Comment by Shrinks99** [comment]
Also ran into this today!  It seems it's hardcoded here: https://github.com/timelinize/timelinize/blob/31c575727c23d088be8d8f106c41160f17b27c90/tlzapp/server.go#L139

Ideally users would be able to set a list of `TRUSTED_HOSTS` as an environment variable which would be passed to both `fillAllowedHosts` and `fillAllowedOrigins`?

**Comment by mholt** [comment]
You can now also set the `TLZ_ORIGIN` env var to a custom origin. However, any `allowed_origins` in the config file would take precedence.
